setup.py

```diff
@@ -1,8 +1,17 @@
 from setuptools import setup
 
 setup(
```

Comment:
Does this break the semantic versioning we gain by using setuptools_scm?

Reply:
I am happy to put this back. The issue I was running into was in using `gcloud build` to build the images which I was then using to run jobs. Gcloud build was failing because when it tars up the source to build the docker image remotely, it was failing due to the lack of the `.git` directory. I'm sure I can find another way around it.

Reply:
Ah, that's interesting, that's a corner case I've never run into. I'd suspect some other Python packages will have worked around this before or something similar.

We could raise an exception and default to some version like astropy does: https://github.com/astropy/astropy/blob/main/astropy/version.py#L4